instrument.py

```diff
@@ -139,6 +143,7 @@ def __init__(self, pxi, expected_root, node: ET.Element = None):
         self.pxi = pxi
         self.expectedRoot = expected_root
         self.enable = False
+        self.isInitialized = False
```

Comment:
same comment for previous casing convention

Reply:
same response for previous comment